src/code/z_scene.c

```diff
@@ -426,7 +426,7 @@ void Scene_CommandAlternateHeaderList(PlayState* play, SceneCmd* cmd) {
             osSyncPrintf("\nげぼはっ！ 指定されたデータがないでええっす！");
 
             if (gSaveContext.sceneLayer == SCENE_LAYER_ADULT_NIGHT) {
-                altHeader = ((SceneCmd**)SEGMENTED_TO_VIRTUAL(cmd->altHeaders.data))[gSaveContext.sceneLayer - 2];
+                altHeader = ((SceneCmd**)SEGMENTED_TO_VIRTUAL(cmd->altHeaders.data))[gSaveContext.sceneLayer - SCENE_LAYER_ADULT_DAY];
```

Comment:
This is just wrong, as it only works in this case by coincidence. Imagine if we added a new `SCENE_LAYER_2`, `SCENE_LAYER_3`, and `SCENE_LAYER_ADULT_DAY` and `SCENE_LAYER_ADULT_NIGHT` were shifted to indexes 4 and 5 for w/e reason. The alt header index calculation would still return 1, mapping to `SCENE_LAYER_2` instead of the intended `SCENE_LAYER_ADULT_DAY`.

Reply:
```suggestion
                altHeader = ((SceneCmd**)SEGMENTED_TO_VIRTUAL(cmd->altHeaders.data))[gSaveContext.sceneLayer - SCENE_LAYER_CHILD_NIGHT - (SCENE_LAYER_ADULT_DAY - SCENE_LAYER_CHILD_NIGHT)];
```
may be more correct,
`- SCENE_LAYER_CHILD_NIGHT` for the sceneLayer to alternate header index,
`- (SCENE_LAYER_ADULT_DAY - SCENE_LAYER_CHILD_NIGHT)` for changing the start index from child_night to adult_day

but tbh this part of the code looks jank and I don't think there's much to do to salvage it

Reply:
`- (SCENE_LAYER_ADULT_DAY - SCENE_LAYER_CHILD_NIGHT)` could also be `- SCENE_LAYER_ADULT_NIGHT + SCENE_LAYER_ADULT_DAY` since the whole purpose of this block is to use adult_day instead of adult_night

Reply:
I would go with `- SCENE_LAYER_ADULT_NIGHT + SCENE_LAYER_ADULT_DAY` like Dragon suggested, since the point is to get the alt header for `SCENE_LAYER_ADULT_DAY` and this is right after a `layer == SCENE_LAYER_ADULT_NIGHT` (so it should arguably just be `[SCENE_LAYER_ADULT_DAY]`)